### DISCUSSION TITLE
useDidUpdate vs useUpdateEffect

### DISCUSSION THREAD

**Discussion by JulienDemarque** [imbhargav5/rooks#513] [Q&A]
I don't really get the difference between useDidUpdate vs useUpdateEffect. Can someone enlighten me?

**Comment by imbhargav5** [comment]
Hi @JulienDemarque . Apologies for the confusion. They are the same hooks. We are trying to streamline the names of the hooks and move away from lifecycle names like `didUpdate` and `didMount`. In a future release `useDidUpdate` will be deprecated.
